geode-core/src/distributedTest/java/org/apache/geode/internal/cache/persistence/PersistentRecoveryOrderDUnitTest.java

```diff
@@ -877,10 +877,42 @@ public void testSplitBrain() {
           createReplicateRegion(regionName, getDiskDirs(getVMId()));
         });
         assertThat(thrown).isInstanceOf(ConflictingPersistentDataException.class);
+        assertThat(thrown.getMessage())
+            .contains("was not part of the same distributed system as the local data");
```

Comment:
Typically you should just use `thrown` as the subject for the assertion rather than `thrown.getMessage()`:
```
assertThat(thrown).hasMessageContaining("was not part of the same distributed system as the local data");
```
That way you can chain together multiple details:
```
assertThat(thrown)
    .isInstanceOf(ConflictingPersistentDataException.class)
    .hasMessageContaining("was not part of the same distributed system as the local data");
```
...and you could even check `hasCauseInstanceOf` or other options.